src/game/entities.cpp

```diff
@@ -1254,12 +1254,18 @@ namespace entities
         if(!m_edit(game::gamemode) && suicide) game::suicide(d, HIT_SPAWN);
     }
 
-    void editent(int i)
+    void editent(int i, bool local)
     {
         extentity &e = *ents[i];
-        fixentity(i, true);
-        if(m_edit(game::gamemode) && game::player1.state == CS_EDITING)
-            client::addmsg(N_EDITENT, "ri5iv", i, (int)(e.o.x*DMF), (int)(e.o.y*DMF), (int)(e.o.z*DMF), e.type, e.attrs.length(), e.attrs.length(), e.attrs.getbuf()); // FIXME
+        if (local)
+        {
+            fixentity(i, true);
+            if(m_edit(game::gamemode) && game::player1.state == CS_EDITING)
+            {
+                client::addmsg(N_EDITENT, "ri5iv", i, (int)(e.o.x*DMF), (int)(e.o.y*DMF), (int)(e.o.z*DMF), e.type, e.attrs.length(), e.attrs.length(), e.attrs.getbuf()); // FIXME
```

Comment:
C-style cast should be avoided. Since #59 we can use STL-compatible methods for vector in most cases.